packages/design-system-docs/src/pages/components/Dialog/dialog-size.example.html

```diff
@@ -30,9 +30,26 @@
       <header class="ds-c-dialog__header" role="banner">
         <h1 class="ds-h2" id="dialog-heading">Dialog heading</h1>
         <button
-          class="dialog-close-btn ds-c-button ds-c-button--transparent ds-c-dialog__close"
```

Comment:
I agree that this should probably go, but it's referenced by some commented out JavaScript at the bottom of this file. I don't know the origin of the code, but I imagine it's helpful to show people how you make it interactive?

Reply:
I added it back!